setup.sh

```diff
@@ -97,72 +97,104 @@ function _inspect
-  help: Show this help dialogue
-
+  echo -e "\e[35mSETUP\e[31m(\e[93m1\e[31m)
+
+\e[38;5;214mNAME\e[39m
+    ${SCRIPT:-${0}} - docker-mailserver administration script
+
+\e[38;5;214mSYNOPSIS\e[39m
+    ./${SCRIPT:-${0}} [ OPTIONS\e[31m...\e[39m ] OBJECT COMMAND [ help \e[31m|\e[39m ARGUMENTS\e[31m...\e[39m ]
+
+    OBJECT \e[31m:=\e[39m { email \e[31m|\e[39m alias \e[31m|\e[39m quota \e[31m|\e[39m config \e[31m|\e[39m relay \e[31m|\e[39m debug }
+
+\e[38;5;214mDESCRIPTION\e[39m
+    This is the main administration script that you use for all interactions with your
+    mail server. Setup, configuration and much more is done with this script.
+
+    Please note that the script executes most of the commands inside the container itself.
+    If the image was not found, this script will pull the \e[37m:latest\e[39m tag of
+    \e[37mmailserver/docker-mailserver\e[39m. This tag refers to the latest release, see the
+    tagging convention in the README under
+    \e[34mhttps://github.com/docker-mailserver/docker-mailserver/blob/master/README.md\e[39m
+
+    You will be able to see detailed information about the script you're invoking and
+    its arguments by appending \e[37mhelp\e[39m after your command. Currently, this
+    does not work with all scripts.
+
+\e[38;5;214mOPTIONS\e[39m
+    \e[94mConfig path, container or image adjustments\e[39m
+        -i IMAGE_NAME
+            Provides the name of the docker-mailserver image. The default value is
+            \e[37mdocker.io/mailserver/docker-mailserver:latest\e[39m
+
+        -c CONTAINER_NAME
+            Provides the name of the running container.
+
+        -p PATH
+            Provides the config folder path. The default is
+            \e[37m${CDIR}/config/\e[39m
+    \e[94mSELinux\e[39m
+        -z
+            Allows container access to the bind mount content that is shared among
+            multiple containers on a SELinux-enabled host.
+
+        -Z
+            Allows container access to the bind mount content that is private and
+            unshared with other containers on a SELinux-enabled host.
+
+    \e[94mOthers\e[39m
+        -h \e[31m|\e[39m help
+            Shows this help dialogue.
+
+\e[38;5;214mCOMMAND\e[39m
+    \e[94mOBJECT\e[39m email \e[31m:=\e[39m
+        ${0} email add <EMAIL ADDRESS> [<PASSWORD>]
+        ${0} email update <EMAIL ADDRESS> [<PASSWORD>]
+        ${0} email del [ ARGUMENTS\e[31m...\e[39m ] <EMAIL ADDRESS>
+        ${0} email restrict <add\e[31m|\e[39mdel\e[31m|\e[39mlist> <send\e[31m|\e[39mreceive> [<EMAIL ADDRESS>]
+        ${0} email list
+
+    \e[94mOBJECT\e[39m alias \e[31m:=\e[39m
+        ${0} alias add <EMAIL ADDRESS> <RECIPIENT>
+        ${0} alias del <EMAIL ADDRESS> <RECIPIENT>
+        ${0} alias list
+
+    \e[94mOBJECT\e[39m quota \e[31m:=\e[39m
+        ${0} quota set <EMAIL ADDRESS> [<QUOTA>]
+        ${0} quota del <EMAIL ADDRESS>
+
+    \e[94mOBJECT\e[39m config \e[31m:=\e[39m
+        ${0} config dkim [ ARGUMENTS\e[31m...\e[39m ]
```

Comment:
Here I am missing an explanation which `ARGUMENTS` are possible. Or was your intention to offload this to the usage function of the called script?

Reply:
Yes. I would not like to state them here, but only indicate that there are arguments. The reason being is that if all commands have args, this overview will become confusing.